qi-lib/flow/core/deforest.rkt

```diff
@@ -459,13 +459,13 @@
                  (loop (op value acc) state)))
          state))))
 
-  (define-inline (cad*r-cstream-next init-countdown next ctx src)
+  (define-inline (cad*r-cstream-next init-countdown name next ctx src)
     (λ (state)
       (let loop ([state state]
                  [countdown init-countdown])
         ((next (λ () ((contract (-> pair? any)
                                 (λ (v) v)
-                                'cad*r-cstream-next ctx #f
+                                name ctx #f
```

Comment:
I think I like this slightly better, but let's still put it in the first commit
in the series (488fd48 (Deforest cadr, caddr, cadddr, caddddr, cadddddr.,
2024-03-11)).

In fact, this makes me think that that `cad*r-datum` should come with a `name`
attribute to pass along here.